### PR TITLE
Fix start round bug

### PR DESCRIPTION
**Pull request by pygmychameleon**
…ng character positions

### CODE REVIEW

File: client/src/scenes/Battle.ts [hunk 1]
```diff
@@ -96,6 +98,10 @@ export default class Battle extends Phaser.Scene {
         }
     }
 
+    private addRoundText() {
+        // this.roundText = this.add.text(300, 200, 'Round ' + this.room.state.currentRound, { fontSize: '30px' }).setScrollFactor(0);
```
Comment (yongning0310) [inline client/src/scenes/Battle.ts:102]: not too sure whats the purpose of addRoundText

File: server/src/rooms/BattleRoom.ts [hunk 2]
```diff
@@ -84,8 +96,8 @@ export class BattleRoom extends Room<MyRoomState> {
     const player = new Player();
 
     // place Player at a random position
```
Comment (yongning0310) [inline server/src/rooms/BattleRoom.ts]: the comment seems a bit misleading, i suppose its fixed position?

File: server/src/rooms/BattleRoom.ts [hunk 3]
```diff
@@ -15,7 +15,9 @@ export class BattleRoom extends Room<MyRoomState> {
   roundCount = 1;
```
Comment (yongning0310) [inline server/src/rooms/BattleRoom.ts:15]: dont seem to check for whether the current round is the last round, roundCount dont seem to be incrementing as well, i suppose this is for future implementation?

File: client/src/scenes/Battle.ts (hunk 1 above)
Comment (shashahchk) [reply client/src/scenes/Battle.ts:102]: this is for future implementation, i will make use of schema field directly.

File: server/src/rooms/BattleRoom.ts (hunk 2 above)
Comment (shashahchk) [reply server/src/rooms/BattleRoom.ts]: my bad forgot to remove this comment.

File: server/src/rooms/BattleRoom.ts (hunk 3 above)
Comment (shashahchk) [reply server/src/rooms/BattleRoom.ts:15]: roundCount is incremented in startRound(). But I plan on using schema in later implementation

**Pull request merged**